client/src/boot/cms/registerTransformations.js

```diff
@@ -0,0 +1,9 @@
+import Injector from 'lib/Injector';
+import WithSudoMode from 'containers/SudoMode/SudoMode';
+
+export default () => {
+  // Apply sudo mode to the mfa list field
+  Injector.transform('apply-sudo-mode-to-mfa', (updater) => {
+    updater.component('RegisteredMFAMethodListField', WithSudoMode);
+  });
```

Comment:
Previously this was done by security-extensions... it doesn't make sense to do that in silverstripe/admin though. This module should register its own transformations.